app/src/main/java/com/fstyle/structure_android/data/source/local/UserLocalDataSource.java

```diff
@@ -7,9 +7,13 @@
 import com.fstyle.structure_android.data.model.User;
 import com.fstyle.structure_android.data.source.UserDataSource;
 import com.fstyle.structure_android.data.source.local.config.sqlite.UserDbHelper;
+import io.reactivex.BackpressureStrategy;
 import io.reactivex.Completable;
 import io.reactivex.CompletableEmitter;
 import io.reactivex.CompletableOnSubscribe;
+import io.reactivex.Flowable;
+import io.reactivex.FlowableEmitter;
+import io.reactivex.FlowableOnSubscribe;
```

Comment:
ERROR: Unused import: io.reactivex.FlowableOnSubscribe. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports